pkg/metricscollector/opentelemetry.go

```diff
@@ -246,3 +246,28 @@ func getScalerMeasurementOption(namespace string, scaledObject string, scaler st
 		attribute.Key("metric").String(metric),
 	)
 }
+
+func (o *OtelMetrics) RecordScalerPaused(namespace string, scaledObject string, active bool) {
+	activeVal := 0
+	if active {
+		activeVal = 1
+	}
+
+	opt := api.WithAttributes(
+		attribute.Key("namespace").String(namespace),
+		attribute.Key("scaledObject").String(scaledObject),
+	)
+
+	cback := func(ctx context.Context, obsrv api.Float64Observer) error {
+		obsrv.Observe(float64(activeVal), opt)
+		return nil
+	}
+	_, err := meter.Float64ObservableGauge(
+		"keda.scaledobject.paused.total",
+		api.WithDescription("Total number of scaled objects currently in a paused state"),
```

Comment:
We shouldn't record "Total number", but mark whether the current ScaledObject is paused - do it the very same way like ScaledObject Activity metrics are done.